salt/states/pip_state.py

```diff
@@ -96,7 +96,21 @@ def pip_has_exceptions_mod(ver):
     HAS_PIP = True
 except ImportError:
     HAS_PIP = False
+    # Remove references to the loaded pip module above so reloading works
+    import sys
```

Comment:
I will try and see this again tomorrow from my laptop, but I think @dwoz refactored this.

Reply:
bump @s0undt3ch did you find what you were referring to here?

Reply:
This is already his code. Carry on. Sorry for the wait.